### DISCUSSION TITLE
[星际统治者2]已开源, 请增加对应游戏

### DISCUSSION THREAD

**Discussion by uiosun** [Gamuxorg/bbs#372] [投递和请求游戏]
StarRuler2-Source: [Github 开源地址](https://github.com/BlindMindStudios/StarRuler2-Source)

我不会打包, 所以木有办法...

顺便一提: 我正在用业余时间做这款游戏的汉化工作 ([相关 issues](https://github.com/BlindMindStudios/StarRuler2-Source/issues/40)), 预计一个月内能完成全部汉化文本, 到时候能更新一下?

**Comment by shouhuanxiaoji** [comment]
OK

**Comment by shouhuanxiaoji** [comment]
已增加 增加的是gog版本的2.0.2

**Comment by uiosun** [comment]
@shouhuanxiaoji 感谢。
